### PR TITLE
Cart management

### PR DESCRIPTION
**Pull request by stutisk**
https://user-images.githubusercontent.com/48529714/154820589-5fe21a4b-a7ba-4335-90fe-e7b6514da4f1.mp4

### CODE REVIEW

**Review by FarhanMobashir — APPROVED** [review]
code is clean. can be merged

File: cart management/cart.html [hunk 1]
```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HomePage</title>
```
Comment (MaruthiPothuganti) [inline cart management/cart.html:8]: Title can be changed as MisTo | Cart or just Cart Maybe..

**Review by Jia-A — APPROVED** [review]
Looks good to me. Can be merged

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested a change. Everything else looks good

**Pull request merged**